HW1/P5/P5.txt

```diff
@@ -1 +1,130 @@
-# Your discussion here
+Path search #1: Kevin Bacon >> Harvard University
+All shortest paths:
+[u'Kevin_Bacon', u'John_Lithgow', u'Harvard_University']
+[u'Kevin_Bacon', u'Six_degrees_of_separation', u'Harvard_University']
+[u'Kevin_Bacon', u'Marisa_Silver', u'Harvard_University']
+[u'Kevin_Bacon', u'College_Bowl', u'Harvard_University']
+[u'Kevin_Bacon', u'Time_(magazine)', u'Harvard_University']
+
+Path search #2: Harvard University >> Kevin Bacon
```

Comment:
Nice, this looks correct. I don't think you had to include every path, but it is pretty cool that you did!